pandas/sparse/frame.py

```diff
@@ -302,7 +302,7 @@ def fillna(self, value=None, method=None, axis=0, inplace=False,
     # ----------------------------------------------------------------------
     # Support different internal representation of SparseDataFrame
 
-    def _sanitize_column(self, key, value):
+    def _sanitize_column(self, key, value, broadcast=False):
```

Comment:
I added this in because insert can call _sanitize_column for DataFrames and SparseDataFrames. If the signatures of each _sanitize_column definition don't match, an error gets thrown. Do you know of a cleaner workaround?